deepcell/graph_utils.py

```diff
@@ -0,0 +1,274 @@
+# Copyright 2016-2019 The Van Valen Lab at the California Institute of
+# Technology (Caltech), with support from the Paul Allen Family Foundation,
+# Google, & National Institutes of Health (NIH) under Grant U24CA224309-01.
+# All rights reserved.
+#
+# Licensed under a modified Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.github.com/vanvalenlab/deepcell-tf/LICENSE
+#
+# The Work provided may be used for non-commercial academic purposes only.
+# For any other use of the Work, including commercial use, please contact:
+# vanvalenlab@gmail.com
+#
+# Neither the name of Caltech nor the names of its contributors may be used
+# to endorse or promote products derived from this software without specific
+# prior written permission.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Utilities for graphs and GCNs"""
+
+# import os
+
+import numpy as np
+import scipy.sparse as sp
+
+from scipy.stats import mode
+from scipy.spatial.distance import cdist
+
+from skimage.io import imread
+from skimage.measure import regionprops_table
+
+
+def get_max_cells(label_image):
+    """ Compute the maximum number of cells in a single batch for a label image
+
+    Args:
+        label_image (int): Label image where every cell has been given a
+            unique integer id
+
+    Returns:
+        np.int: The maximum number of cells in a single batch of the
```

Comment:
I think its just a regular `int`.
```suggestion
        int: The maximum number of cells in a single batch of the
```